corgi/api/views.py

```diff
@@ -114,6 +113,19 @@ def list(self, request):
         with connection.cursor() as cursor:
             cursor.execute("SELECT pg_size_pretty(pg_database_size(current_database()));")
             db_size = cursor.fetchone()
+            cursor.execute(
+                "SELECT reltuples AS estimate FROM pg_class "
+                "WHERE relname = 'core_productcomponentrelation';"
+            )
+            pcr_count = cursor.fetchone()
+            cursor.execute(
+                "SELECT reltuples AS estimate FROM pg_class WHERE relname = 'core_component';"
+            )
+            component_count = cursor.fetchone()
+            cursor.execute(
+                "SELECT reltuples AS estimate FROM pg_class WHERE relname = 'core_softwarebuild';"
+            )
+            sb_count = cursor.fetchone()
```

Comment:
Can you add a comment in the code please explaining why these are done using estimates / raw SQL?